tpm2/test/evict_control_test.go

```diff
@@ -0,0 +1,38 @@
+package tpm2test
+
+import (
+	. "github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/tpm2/transport/simulator"
+	"testing"
+)
+
+func TestEvictControl(t *testing.T) {
+	thetpm, err := simulator.OpenSimulator()
+	if err != nil {
+		t.Fatalf("could not connect to TPM simulator: %v", err)
+	}
+	defer thetpm.Close()
+
+	srkCreate := CreatePrimary{
+		PrimaryHandle: TPMRHOwner,
+		InPublic:      New2B(ECCSRKTemplate),
+	}
+
+	srkCreateRsp, err := srkCreate.Execute(thetpm)
+	if err != nil {
+		t.Fatalf("could not generate SRK: %v", err)
+	}
+
+	evictControlRsp, err := EvictControl{
```

Comment:
`_, err := EvictControl` would allow you to delete line 37 below (`_ = evictControlRsp`)